internal/certificateprovider/certificateproviderpage/enter_reference_number.go

```diff
@@ -55,14 +86,18 @@ func EnterReferenceNumber(tmpl template.Template, shareCodeStore ShareCodeStore,
 				})
 
 				if _, err := certificateProviderStore.Create(ctx, shareCode, session.Email); err != nil {
-					return err
+					return fmt.Errorf("error creating certificate provider: %w", err)
 				}
 
 				appData.LpaID = shareCode.LpaKey.ID()
 				return certificateprovider.PathWhoIsEligible.Redirect(w, r, appData, appData.LpaID)
 			}
 		}
 
+		if results.Empty() {
```

Comment:
also we won't want this

Reply:
Linked with the above comment - it was the simplest way to hide the nav for users with no other LPAs in their account.